### PR TITLE
Update licenses

### LINKED ISSUE #1947: Update Licenses for 1.2 Release
* License files for Operator
 * License files for Executor

### PR DESCRIPTION
**Pull request by cliveseldon**
Fixes #1947

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **cliveseldon**

If they are not already assigned, you can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Mon Jun 15 17:51:34 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=2

**Comment by seldondev** [comment]
Mon Jun 15 17:51:32 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=1

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 30556c38fcf2e0df7543a52d2c9a2f0f024d933c | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Tue Jun 16 10:12:25 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=3

**Comment by seldondev** [comment]
Tue Jun 16 10:16:41 UTC 2020
The logs for [lint] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=5

**Comment by seldondev** [comment]
Tue Jun 16 10:16:45 UTC 2020
The logs for [pr-build] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=4

File: executor/licenses/repo.txt [hunk 1]
```diff
@@ -1,42 +1,84 @@
...
 census-ecosystem/opencensus-go-exporter-prometheus
 Azure/azure-sdk-for-go
+Azure/go-ansiterm
 Azure/go-autorest
 BurntSushi/toml
+BurntSushi/xgb
 NYTimes/gziphandler
 PuerkitoBio/purell
 PuerkitoBio/urlesc
+Shopify/sarama
```
Comment (adriangonz) [inline executor/licenses/repo.txt:13]: Why is `sarama` also included? Wasn't that part of the Kafka PR?

Comment (cliveseldon) [reply executor/licenses/repo.txt:13]: Interesting. Due to Zipkin.

https://github.com/openzipkin/zipkin-go/blob/ef1b34462145c161724adcc13be368269c29019e/go.mod#L4

Comment (cliveseldon) [reply executor/licenses/repo.txt:13]: No it wasn't part of the kafka PR yet.

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1954#issuecomment-644879986" title="Approved">adriangonz</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [adriangonz]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Tue Jun 16 16:55:39 UTC 2020
The logs for [lint] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=7

**Comment by seldondev** [comment]
Tue Jun 16 16:55:40 UTC 2020
The logs for [pr-build] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1954/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1954 --build=6

**Comment by seldondev** [comment]
@cliveseldon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | f33f5848a75a11486c38186324e8fadeb46eba1a | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**